channels/urbdrc/client/libusb/libusb_udevman.c

```diff
@@ -418,13 +433,15 @@ static void udevman_free(IUDEVMAN* idevman)
 		return;
 
 	udevman->running = FALSE;
-	WaitForSingleObject(udevman->thread, INFINITE);
+	if (udevman->thread)
+		WaitForSingleObject(udevman->thread, INFINITE);
 
 	udevman_unregister_all_udevices(idevman);
 	CloseHandle(udevman->devman_loading);
 	CloseHandle(udevman->thread);
```

Comment:
This should be covered by the `NULL` check too.

Reply:
But CloseHandle already checks for null pointers itself

Reply:
@lukasfink1 ours does, windows is different. (depends on version)